aspnetcore/fundamentals/host/platform-specific-configuration.md

```diff
@@ -161,197 +161,85 @@ Options for hosting startup activation are:
-
-# [Windows](#tab/windows)
-
-Set the `DOTNET_ADDITIONAL_DEPS` environment variable to the following value:
-
-```
-%UserProfile%\.dotnet\x64\additionalDeps\StartupDiagnostics\
-```
-
-# [macOS](#tab/macos)
-
-Set the `DOTNET_ADDITIONAL_DEPS` environment variable to the following value:
-
-```
-/Users/<USER>/.dotnet/x64/additionalDeps/StartupDiagnostics/
-```
-
-# [Linux](#tab/linux)
-
-Set the `DOTNET_ADDITIONAL_DEPS` environment variable to the following value:
-
-```
-/Users/<USER>/.dotnet/x64/additionalDeps/StartupDiagnostics/
-```
-
----
+To be able to activate the enhancement without having package reference to it one should use `additionalDeps` feature of the runtime. It allows to extend applications library graph by providing set of additional `.deps.json` files that would get merged with apps own `.deps.json` on startup and would make hosting startup assembly discoverable and loadable.
+
+Easy way of generating additional deps file is running `dotnet publish` on runtime store manifest file refernced in previous section and then removing manifest reference from libraries and runtime section of resulting deps.json file.
+
+``` diff
+{
+  "runtimeTarget": {
+    "name": ".NETCoreApp,Version=v2.1",
+    "signature": "4ea77c7b75ad1895ae1ea65e6ba2399010514f99"
+  },
+  "compilationOptions": {},
+  "targets": {
+    ".NETCoreApp,Version=v2.1": {
+-      "store.manifest/1.0.0": {
+-        "dependencies": {
+-          "StartupDiagnostics": "1.0.0"
+-        },
+-        "runtime": {
+-          "store.manifest.dll": {}
+-        }
+-      },
+      "StartupDiagnostics/1.0.0": {
+        "runtime": {
+          "lib/netcoreapp2.1/StartupDiagnostics.dll": {
+            "assemblyVersion": "1.0.0.0",
+            "fileVersion": "1.0.0.0"
+          }
+        }
+      }
+    }
+  },
+  "libraries": {
+-    "store.manifest/1.0.0": {
+-      "type": "project",
+-      "serviceable": false,
+-      "sha512": ""
+-    },
+    "StartupDiagnostics/1.0.0": {
+      "type": "package",
+      "serviceable": true,
+      "sha512": "sha512-oiQr60vBQW7+nBTmgKLSldj06WNLRTdhOZpAdEbCuapoZ+M2DJH2uQbRLvFT8EGAAv4TAKzNtcztpx5YOgBXQQ==",
+      "path": "startupdiagnostics/1.0.0",
+      "hashPath": "startupdiagnostics.1.0.0.nupkg.sha512"
+    }
+  }
+}
+```
+
+Deps.json file is then placed into the following directory structure:
+
+```
+<ADDITIONAL_DEPS_PATH>/shared/<SHARED_FRAMEWORK_NAME>/<SHARED_FRAMEWORK_VERSION>/<ENHANCEMENT_ASSEMBLY_NAME>.deps.json
+```
+
+Where `ADDITIONAL_DEPS_PATH` is location that is being added to `DOTNET_ADDITIONAL_DEPS`, `SHARED_FRAMEWORK_NAME` is shared framework required for this additionalDeps file, `SHARED_FRAMEWORK_VERSION` minimum shared framework version.
+
+For runtime to be able to discover runtime store location  needs to be added to `DOTNET_ADDITIONAL_DEPS` environment variable.
```

Comment:
I would make a grammar/style suggestion, but I can't grasp the meaning of the sentence. There's a prep phrase object missing or the subject of the sentence is missing.

Reply:
Hah. What I was trying to say is that you need to add `additionalDeps` location to  `DOTNET_ADDITIONAL_DEPS` environment variable.